shell.nix

```diff
@@ -0,0 +1,12 @@
+with import <nixpkgs> {};
+
+mkShell {
+    buildInputs = [
+        python39
+        python39Packages.poetry
+        ripgrep
+        jq
+        datamash
+        findutils
+    ];
+}
```

Comment:
shows you the system dependencies you need and if you have nix also lets you setup a local shell with them all available by running `nix-shell`